swatch-billable-usage/src/main/java/com/redhat/swatch/billable/usage/admin/api/InternalBillableUsageController.java

```diff
@@ -157,21 +161,33 @@ private BillableUsage toBillableUsage(BillableUsageRemittanceEntity remittance)
         Objects.nonNull(remittance.getStatus())
             ? BillableUsage.Status.fromValue(remittance.getStatus().getValue())
             : null;
-    return new BillableUsage()
-        .withOrgId(remittance.getOrgId())
-        .withTallyId(remittance.getTallyId())
-        .withUuid(remittance.getUuid())
-        .withSnapshotDate(remittance.getRemittancePendingDate())
-        .withProductId(remittance.getProductId())
-        .withSla(BillableUsage.Sla.fromValue(remittance.getSla()))
-        .withUsage(BillableUsage.Usage.fromValue(remittance.getUsage()))
-        .withBillingProvider(
-            BillableUsage.BillingProvider.fromValue(remittance.getBillingProvider()))
-        .withBillingAccountId(remittance.getBillingAccountId())
-        .withMetricId(remittance.getMetricId())
-        .withValue(remittance.getRemittedPendingValue())
-        .withHardwareMeasurementType(remittance.getHardwareMeasurementType())
-        .withStatus(remittanceStatus);
+    var billableUsage =
+        new BillableUsage()
+            .withOrgId(remittance.getOrgId())
+            .withTallyId(remittance.getTallyId())
+            .withUuid(remittance.getUuid())
+            .withSnapshotDate(remittance.getRemittancePendingDate())
+            .withProductId(remittance.getProductId())
+            .withSla(BillableUsage.Sla.fromValue(remittance.getSla()))
+            .withUsage(BillableUsage.Usage.fromValue(remittance.getUsage()))
+            .withBillingProvider(
+                BillableUsage.BillingProvider.fromValue(remittance.getBillingProvider()))
+            .withBillingAccountId(remittance.getBillingAccountId())
+            .withMetricId(remittance.getMetricId())
+            .withHardwareMeasurementType(remittance.getHardwareMeasurementType())
+            .withStatus(remittanceStatus);
+
+    billableUsage.setValue(getBillableValue(remittance, billableUsage));
```

Comment:
let's also set the billingFactor for better traceability like done in BillableUsageService.